app/models/persistent_volume.rb

```diff
@@ -1,5 +1,6 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  include NewWithTypeStiMixin
```

Comment:
But since we are just trying to get the schema in, this can stay as is, and we can tweak in a follow up PR...it's up to you.